lib/kredis/types/expirable.rb

```diff
@@ -0,0 +1,42 @@
+module Kredis::Types::Expirable
+  extend ActiveSupport::Concern
+
+  included do
+    proxying :expire, :expireat, :ttl
+
+    attr_accessor :expires_in, :expires_at
+
+    def expire_in(seconds)
+      expire seconds.to_i
+    end
+
+    def expire_at(datetime)
```

Comment:
If we kept `expire_at` we should review existing cases where we use `expire_in` to support this. E.g: add `:exat` support to `#set` operations using `:ex`. Maybe we can only address `expire_in` in the initial implementation?

I think consistency is important. So if we want to support `expire_at`, I'd make sure it's supported across all types.

Reply:
I'm not a Redis expert, but what if we stopped using `ex` and relied solely on the proposed implementation?

Reply:
We'd lose the atomicity of doing both operations in a single one, but I think it's a fair tradeoff. We can always optimize things later if we find the need to do it.

I have mixed feelings about not doing the expire within a transaction. It can happen that the client loses the connection to redis right before setting the `expire`, letting the key without the expiration date set behind.

Reply:
I agree. I'll start to implement it to all other types, and when #78 gets merged, I can give it a shot for the approach you suggested and start a transaction if we are not in yet. Is it worth it?